languagetool-language-modules/es/src/main/resources/org/languagetool/rules/es/remote-rule-filters.xml

```diff
@@ -459,25 +454,25 @@ To ignore a remote rule match, set the <marker> so that it exactly covers the te
                 <example correction=""><marker>Juan</marker> no va hacia su casa.</example>
                 <example correction=""><marker>Juan</marker> no se va hacia su casa.</example>
                 <example correction=""><marker>Juan</marker> no se lo da.</example>
-                <example>Mañana, alunizará.</example>
-                <example>Gracias, es todo.</example>
-                <example>No, es de segunda mano.</example>
-                <example>Cansado, se fue a la cama antes de lo normal.</example>
-                <example>Tranquilo, es sólo un espantapájaros.</example>
-                <example>Bill, abre la puerta.</example>
-                <example>Jim, cierra la ventana.</example>
-                <example>Segundo, recibe una violenta oposición.</example>
-                <example>Mira, está lloviendo.</example>
-                <example>Louis, vienen tus amigos.</example>
-                <example>Tom, hay algo que quiero darte.</example>
-                <example>Mira, se le rompió la fuente a esa señora.</example>
-                <example>Tom, te llama tu mamá.</example>
-                <example>Mary, venga, deja de protestar.</example>
-                <example>Ricardo, ora por mi.</example>
-                <example>María, me gustan mucho las enumeraciones.</example>
-                <example>Tom, te llama mamá.</example>
-                <example>Claro, le duele.</example>
-                <example>Tom, no es necesario que te disculpes.</example>
+                <example>Mañana alunizará.</example>
+                <example>Gracias es todo.</example>
+                <example>No es de segunda mano.</example>
+                <example>Cansado se fue a la cama antes de lo normal.</example>
+                <example>Tranquilo es sólo un espantapájaros.</example>
+                <example>Bill abre la puerta.</example>
+                <example>Jim cierra la ventana.</example>
+                <example>Segundo recibe una violenta oposición.</example>
+                <example>Mira está lloviendo.</example>
+                <example>Louis vienen tus amigos.</example>
+                <example>Tom hay algo que quiero darte.</example>
+                <example>Mira se le rompió la fuente a esa señora.</example>
+                <example>Tom te llama tu mamá.</example>
+                <example>Mary venga, deja de protestar.</example>
+                <example>Ricardo ora por mi.</example>
+                <example>María me gustan mucho las enumeraciones.</example>
+                <example>Tom te llama mamá.</example>
+                <example>Claro le duele.</example>
+                <example>Tom no es necesario que te disculpes.</example>
```

Comment:
Okay. This means: if the model tries to add a comma to these sentences, the rule in remote-rule-filter.xml will not prevent the model from doing it.